buildpack.md

```diff
@@ -666,6 +666,23 @@ For each file written to `<layers>/<layer>/env.launch/` by `/bin/build`, the lif
 The lifecycle MUST consider the name of the environment variable to be the name of the file up to the first period (`.`) or to the end of the name if no periods are present.
 In all cases, file contents MUST NOT be evaluated by a shell or otherwise modified before inclusion in environment variable values.
 
+##### Override
+
+If the environment variable file name has no period-delimited suffix, then the value of the environment variable MUST be the file contents.
```

Comment:
What do you think about adding an examples area to clearly show the association between the file name and the definition of behavior? I think this would also have the benefit of hoisting up the no period-delimited behavior to make it clearer.

##### Examples

| Filename | Modification Behavior
|------|-------------------------------------------
| `<layers>/<layer>/env/<env_var>`                   | [Override](#delimiter)
| `<layers>/<layer>/env/<env_var>.override`    | [Override](#override)
| `<layers>/<layer>/env/<env_var>.default`      | [Default](#default)
| `<layers>/<layer>/env/<env_var>.delim`         | [Default](#delimiter)
| `<layers>/<layer>/env/<env_var>.prepend`    | [Default](#prepend)
| `<layers>/<layer>/env/<env_var>.append`     | [Default](#append)

Reply:
Good idea. I updated the branch to move all of the suffix information into a table at the top. With that change I thought it made sense to alphabetize the modification types (instead of putting the default first).

Reply:
Are we still going to have a `.override`?  We're already breaking compatibility, and I'm not a fan of having two ways to accomplish the same behavior.

Reply:
By using `.override` or `.prepend` explicitly buildpack authors can currently write buildpacks that will not break when the API changes. This would not be true if we removed `.override` in buildpack API 0.5.

Also, removing `.override` would require more buildpacks to update to accomodate this change.

In my mind, the above reasons outweigh the cost of having two ways to accomplish the same behavior. Personally I also like the symmetry of having a suffix for every operation (but that probably not a very good reason).

Reply:
Why support the no-extension one then?